### PR TITLE
Fix: Types Typo

### PR DESCRIPTION
**Pull request by zusinShinpei**

### CODE REVIEW

File: desktop/doctor/src/index.ts [hunk 1]
```diff
@@ -77,28 +81,27 @@ export function getHealthchecks(): Healthchecks {
...
             const hasProblem = result.hasProblem;
-            const message = hasProblem
-              ? `OpenSSL (https://wiki.openssl.org/index.php/Binaries) is not installed or not added to PATH. ${result.message}.`
-              : `OpenSSL (https://wiki.openssl.org/index.php/Binaries) is installed and added to PATH. ${result.message}.`;
             return {
               hasProblem,
-              message,
+              message: `OpenSSL (https://wiki.openssl.org/index.php/Binaries) is${
+                hasProblem ? ' not' : ''
+              } not installed or not added to PATH. ${result.message}.`,
```
Comment (jknoxville) [inline desktop/doctor/src/index.ts:91]: I don't think this message makes sense now:

When hasProblem:
`OpenSSL is not not installed or not added to PATH.`

When !hasProblem:
`OpenSSL is not installed or not added to PATH.`

File: desktop/doctor/src/index.ts [hunk 2]
```diff
@@ -77,28 +81,27 @@ export function getHealthchecks(): Healthchecks {
...
+          run: async () => {
             const isAvailable = await isWatchmanAvailable();
             return {
               hasProblem: !isAvailable,
-              message: isAvailable
-                ? 'Watchman file watching service (https://facebook.github.io/watchman/) is installed and added to PATH. Live reloading after changes during Flipper plugin development is enabled.'
-                : 'Watchman file watching service (https://facebook.github.io/watchman/) is not installed or not added to PATH. Live reloading after changes during Flipper plugin development is disabled.',
+              message: `Watchman file watching service (https://facebook.github.io/watchman/) is${
+                isAvailable ? '' : ' not'
+              } installed or not added to PATH. Live reloading after changes during Flipper plugin development is disabled.`,
```
Comment (jknoxville) [inline desktop/doctor/src/index.ts:104]: Same here.

**Review by mweststrate — CHANGES_REQUESTED** [review]
Hi @zusinShinpei, do you still intend to follow up on the review comments? If not, then best close the PR. Thanks!

**Comment by mweststrate** [comment]
Closing for inactivity

**Pull request closed without merging**